utils/rpms/daos.spec

```diff
@@ -9,7 +9,7 @@ Version:       1.1.2.1
 Release:       6%{?relval}%{?dist}
 Summary:       DAOS Storage Engine
 
-License:       Apache
+License:       BSD-2-Clause-Patent
```

Comment:
You need to bump the release on line 9 and add a change log entry

Reply:
Oops, yes.